src/cli/shared/__tests__/getConfig.spec.js

```diff
@@ -156,6 +156,25 @@ describe('Joi validate', () => {
     spy.mockRestore();
   });
 
+  it('should throw error if validate failed', () => {
+    const { MOCK_FILE_WITH_PLATFORM } = setup();
+    MOCK_FILE_WITH_PLATFORM.slack.accessToken = 12345;
```

Comment:
writing a comment will enhance readability, just like `// accessToken need to be string type`